lib/tasks/import_datastream.rake

```diff
@@ -26,7 +26,11 @@ namespace :ssg do
   desc 'Update compliance DB with the supported SCAP Security Guide versions'
   task import_rhel_supported: [:environment] do
     # DATASTREAM_FILENAMES from openscap_parser's ssg:sync
-    ENV['DATASTREAMS'] = ::SupportedSsg.available_upstream.map do |ssg|
+    ENV['DATASTREAMS'] = ::SupportedSsg.available_upstream.sort_by do |ssg|
+      Gem::Version.new(ssg.version)
+    end.reverse
+
+    ENV['DATASTREAMS'].map do |ssg|
```

Comment:
You either want `map!` or just put the `map` after the `.reverse`

Reply:
Oops! haha good catch. I should test after I refactor for rubocop :see_no_evil:

Reply:
Alright, fixed. I just introduced a new variable. Rubocop didn't like chaining blocks.